.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,13 @@ the linked issue.
 -->
 
 Related to #
+
+#### Things to check
+
+Here are some important things to check on each PR as an author and reviewer.
```

Comment:
What do you think of making the language here even more direct about timing and responsibilities?

Something like...

```markdown
#### Pre-merge Checklist

Complete all of the items on this checklist **before** merging the PR into trunk. Both the PR author and review are responsible for ensuring the checklist is completed. Not all items will apply to every PR, and any inapplicable items can just be marked as complete.
```